CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ project(realm)
 
 # Project-wide build flags
 set(CMAKE_CXX_STANDARD 20)
+#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address")
```

Comment:
Why is this commented out?

Reply:
Because it should only be on for testing.